src/main/js/src/components/UI/RadioButton/RadioButton.js

```diff
@@ -7,7 +7,7 @@ const radioButton = props => (
   <div
     className={[
       classes.RadioButton,
-      props.disabled ? classes.RadioButtonDisabled : '',
+      !props.disabled ? classes.RadioButtonEnabled : classes.RadioButtonDisabled,
```

Comment:
To make it clearer
- the exclamation mark can be removed
- if-else conditions can be reversed